app/commands/submission/representation/create.rb

```diff
@@ -4,14 +4,17 @@ class Submission::Representation::Create
   initialize_with :submission, :tooling_job, :ast_digest, :exercise_representer_version
 
   def call
-    Submission::Representation.create!(
+    representation = Submission::Representation.find_create_or_find_by!(
       submission:,
-      tooling_job_id: tooling_job.id,
       ops_status: tooling_job.execution_status.to_i,
```

Comment:
Do we also want to include the `ops_status` here?

Reply:
Yeah, I think so. If the previous was a 400 and now this is a 200, we want a new record. Good question though - I pondered it too.